Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -479,15 +482,13 @@ void USpatialReceiver::OnComponentUpdate(Worker_ComponentUpdateOp& Op)
 		return;
 	}
 
-	if (Op.update.component_id == Info->SingleClientComponent)
-	{
-		UObject* TargetObject = GetTargetObjectFromChannelAndClass(ActorChannel, Class);
-		ApplyComponentUpdate(Op.update, TargetObject, ActorChannel);
-	}
-	else if (Op.update.component_id == Info->MultiClientComponent)
+	if (Op.update.component_id == Info->SingleClientComponent || Op.update.component_id == Info->MultiClientComponent)
 	{
 		UObject* TargetObject = GetTargetObjectFromChannelAndClass(ActorChannel, Class);
-		ApplyComponentUpdate(Op.update, TargetObject, ActorChannel);
+		if(TargetObject != nullptr)
```

Comment:
Nit: space after `if`

Reply:
Also want to log if `TargetObject` is missing.

Reply:
Move TargetObject definition into if statement.